wireless_modules/middle_module/README.md

```diff
@@ -0,0 +1,62 @@
+# DAS - MIDDLE WIRELESS MODULE
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors)
+
+The Middle Wireless Module allows us to collect and send data from the sensors attached to the bike. The middle wireless module consists of a micro controller (ESP32 currently) which gathers the data from the on board sensors and sends them using MQTT (over the internet).
+
+This repository contains all the code related to Monash Human Power's middle wireless module - part of their Data Acquisition System (DAS).
+
+## Getting Started
+
+1. Press clone on the root folder and use it to clone the entire repository on the command line.
+2. Create a ``config.py`` file in the same directory as the contents of this folder (refer below).
```

Comment:
Might have been a misunderstanding - you only need a single backtick on each side of the text you want to make monospace. `` `config.py` `` becomes `config.py`